server/document-watch.worker.js

```diff
@@ -14,6 +14,14 @@ function postEvent(type, data = {}) {
 }
 
 function postDocumentInfo(filePath, changeType) {
+  // Note! On Windows, `glob.sync()` will return URLs that always use
+  // the `/` character (POSIX STYLE, https://www.npmjs.com/package/glob#windows)
+  // But the `Document.urlToFolderPath()` function will respect Windows
+  // notation and use `\` characters.
+  // Now, when we use it, it'll work as if `glob.sync()` and returned paths
+  // in a fashion that is expected in the OS.
+  filePath = path.split(filePath).join(path.sep);
```

Comment:
What do you think about this solution @fiji-flo ?

Reply:
I think you need and explicit `path.join(filePath.split("/"))`

I assume `path.split` splits only on `path.sep` which would be `\` in this case but we wanna split `filePath` on `/`, don't we?

Reply:
Did you literally push my proposed change while I was writing this? Spooky